bref/Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM bref/php-80-fpm-dev:1.0.1
-USER root
+FROM alpine:3.12 as dependencies
+RUN apk add --no-cache curl
+RUN curl -SsL https://github.com/ldn-softdev/jtc/releases/download/LatestBuild/jtc-linux-64.latest > /usr/local/bin/jtc && \
+  chmod 755 /usr/local/bin/jtc && \
+  jtc -help
 RUN curl -SsL https://github.com/kvz/json2hcl/releases/download/v0.0.6/json2hcl_v0.0.6_linux_amd64 > /usr/local/bin/json2hcl && \
   chmod 755 /usr/local/bin/json2hcl && \
   json2hcl -version
-RUN curl --compressed -SsL https://github.com/antonmedv/fx/releases/download/20.0.2/fx-linux.zip -o - | \
-  gunzip - > /usr/local/bin/fx && \
-  chmod 755 /usr/local/bin/fx
+
+FROM bref/php-80-fpm-dev:latest
```

Comment:
prior to merge, switch back on a bref tagged image (eg: `bref/php-80-fpm-dev:1.0.2`)

Reply:
we wait for merge of https://github.com/brefphp/bref/pull/830#discussion_r551980075